### ISSUE TITLE
CircleCI deploy issue

### ISSUE THREAD

**Issue by rgommers** [Quansight-Labs/quansight-labs-site#10]
One autodeploy issue left:

```
#!/bin/bash -eo pipefail
git config --global user.name 'CirclCI'
git config --global user.email 'circleci@invalid'
git config --global push.default 'simple'
if [ "${CIRCLE_BRANCH}" == "master" ]; then
  nikola github_deploy -m 'Nikola auto deploy [ci skip]'
fi

Scanning posts........done!
Scanning posts........done!
[2019-04-18T21:58:16Z] INFO: github_deploy: ==> ['git', 'checkout', 'src']
error: pathspec 'src' did not match any file(s) known to git.
[2019-04-18T21:58:16Z] INFO: github_deploy: ==> ['git', 'checkout', '-b', 'src']
Switched to a new branch 'src'
[2019-04-18T21:58:16Z] INFO: github_deploy: ==> ['git', 'add', '.']
[2019-04-18T21:58:16Z] INFO: github_deploy: ==> ['git', 'diff-index', '--quiet', 'HEAD']
[2019-04-18T21:58:16Z] INFO: github_deploy: ==> ['git', 'commit', '-am', 'Nikola auto deploy [ci skip]\n\nNikola version: 8.0.2']
[src aac97c8] Nikola auto deploy [ci skip]
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 .doit.db
[2019-04-18T21:58:16Z] INFO: github_deploy: ==> ['ghp-import', '-n', '-m', 'Nikola auto deploy [ci skip]\n\nSource commit: aac97c88c459a42f90c653463c0e6b58d9e4fe35\nNikola version: 8.0.2', '-p', '-r', 'origin', '-b', 'gh-pages', 'output']
Warning: Permanently added the RSA host key for IP address '192.30.253.113' to the list of known hosts.

ERROR: The key you are authenticating with has been marked as read only.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Traceback (most recent call last):
  File "/usr/local/bin/ghp-import", line 10, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/ghp_import/cli.py", line 198, in main
    sp.check_call(['git', 'push', opts.remote, opts.branch])
  File "/usr/local/lib/python3.7/subprocess.py", line 347, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['git', 'push', 'origin', 'gh-pages']' returned non-zero exit status 128.
[2019-04-18T21:58:16Z] ERROR: github_deploy: Failed GitHub deployment -- command ['ghp-import', '-n', '-m', 'Nikola auto deploy [ci skip]\n\nSource commit: aac97c88c459a42f90c653463c0e6b58d9e4fe35\nNikola version: 8.0.2', '-p', '-r', 'origin', '-b', 'gh-pages', 'output'] returned 1
```
@hameerabbasi your last commit on `.circleci/config.yml` said "move changed to ENV vars", however env vars in Settings are empty and there is no comment with what ENV var to add. Did this work before?

**Comment by hameerabbasi** [comment]
Fixed in #12, sorry for the noise.

**Comment by rgommers** [comment]
awesome, thanks for the help @hameerabbasi
